github-action-build/setup.sh

```
@@ -82,25 +84,25 @@ echo '======================= Installing WSO2 IS Accelerator Updates ===========
 name=$(echo "$WSO2_USERNAME" | cut -d'@' -f1)
 WSO2_UPDATES_HOME=home/$name/.wso2updates
 sudo mkdir -p /home/$name/.wso2-updates/docker && sudo chmod -R 777 /home/$name/.wso2-updates
-cp ${RUNNER_HOME}/github-action-build/wso2update_linux $TEST_HOME/wso2is-6.1.0/wso2-obiam-accelerator-3.0.0/bin/
-chmod +x $TEST_HOME/wso2is-6.1.0/wso2-obiam-accelerator-3.0.0/bin/wso2update_linux
-$TEST_HOME/wso2-obiam-accelerator-3.0.0/wso2is-6.1.0/bin/wso2update_linux -v --username $WSO2_USERNAME --password $WSO2_U2_PASSWORD ||  ($TEST_HOME/wso2-obiam-accelerator-3.0.0/wso2is-6.1.0/bin/wso2update_linux -v --username $WSO2_USERNAME --password $WSO2_U2_PASSWORD )
+cp ${RUNNER_HOME}/github-action-build/wso2update_linux $TEST_HOME/wso2is-6.1.0/wso2-obiam-accelerator-3.3.0/bin/
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)